README.md

```diff
@@ -136,12 +136,21 @@ See [docs/vagrant.md](docs/vagrant.md)
 
 * Go >= 1.16
 * LLVM/clang >= 1.12
+* Bison 
+* Lex/Flex >= 2.5.31
 
 ### Building
 
+
+Build libpcap before building pwru:
```

Comment:
I think this is not needed, as the dependency https://github.com/cilium/pwru/blob/main/Makefile#L17 should trigger the libpcap build process.

Reply:
You're right.